src/js/keyboardInput.js

```diff
@@ -226,4 +240,63 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }
     };
 
+    fluid.defaults("gpii.firstDiscovery.keyboardInputTts", {
```

Comment:
We should supply good documentation here about which grade this mixin is an "addon" for and what capabilities it supplies.